Configuration/scripts/makeEfficiencyPlots.py

```diff
@@ -262,11 +266,24 @@ def MakeOneHist(dirName, histogramName):
         if arguments.normalizeToUnitArea:
             yAxisLabel = yAxisLabel + " (Unit Area Norm.)"
 
-        #Histogram = ROOT.TGraphAsymmErrors(NumHistogramObj,DenHistogramObj)
-        if arguments.noTGraph or Histogram.Class().InheritsFrom("TH2"):
+        #check if bin content is consistent
+        for i in range(1,Histogram.GetNbinsX()+1):
```

Comment:
I think this should be more verbose if failed, the user should know their efficiency is odd. You might use `TEfficiency::CheckConsistency` or `CheckBinning` which come with info printouts.

Reply:
done